Makefile

```diff
@@ -93,22 +97,22 @@ $(LINUX_RELEASE): $(LINUX_BIN)
 	#upx $(DARWIN_RELEASE)
 	#$ (call write-signature,$(DARWIN_RELEASE))
 
-#WINDOWS_RELEASE = $(RELEASE_DIR)/$(notdir $(WINDOWS_BIN))-$(VERSION).exe
-#$(WINDOWS_RELEASE): $(WINDOWS_BIN)
-	#cp bin/$(WINDOWS).exe $(WINDOWS_RELEASE)
-	#upx $(WINDOWS_RELEASE)
-	#\$ (call write-signature,$(WINDOWS_RELEASE))
-	#osslsigncode sign \
-		#-pkcs12 $(CODE_SIGNING_CERT) \
-		#-h sha1 \
-		#-n "Dividat Driver" \
-		#-i "https://www.dividat.com/" \
-		#-in $(WINDOWS_RELEASE) \
-		#-out $(WINDOWS_RELEASE)
+WINDOWS_RELEASE = $(RELEASE_DIR)/dividat-driver-windows-amd64-$(VERSION).exe
+$(WINDOWS_RELEASE): $(WINDOWS_BIN)
+	cp $(WINDOWS_BIN) $(WINDOWS_RELEASE)
+	upx $(WINDOWS_BIN)
+	osslsigncode sign \
```

Comment:
Timestamp missing. driverv0 should give details to use.

Further information:
- https://docs.microsoft.com/en-us/windows/desktop/seccrypto/time-stamping-authenticode-signatures#about-cryptographic-time-stamping
- https://github.com/antoinevg/osslsigncode/blob/master/README#L73-L76